src/main/java/htsjdk/variant/variantcontext/writer/BCF2Writer.java

```diff
@@ -221,6 +221,11 @@ public void close() {
         super.close();
     }
 
+    @Override
+    public void setVcfHeader(VCFHeader header) {
+        //no-op
```

Comment:
Why is this a "no-op"? This doesn't seem right. The `add()` method in `BCF2Writer` will blow up without a header, since it calls `vc = vc.fullyDecode(header, false);`